### PR TITLE
Add easy HRP countries

### PR DESCRIPTION
**Pull request by turnerm**
Here are the "easy" HRP countries (without strange COD AB issues mentioned [here](https://github.com/OCHA-DAP/ocha-anticipy/issues/181). 

Opening as a draft because I still have to test everything myself!

### CODE REVIEW

File: src/ochanticipy/config/countries/lby.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+iso3: lby
+codab:
+  hdx_dataset_name: lby_adm_unosat_lbsc_20180507_SHP.zip
+  layer_base_name: lby_adm_unosat_lbsc_20180507_SHP/lby_admbnda_adm{admin_level}_unosat_lbsc_20180507.shp
+  admin_level_max: 2
+usgs_ndvi:
+  area_name: east-africa
```
Comment (caldwellst) [inline src/ochanticipy/config/countries/lby.yaml:7]: Should be North Africa.

**Review by caldwellst — APPROVED** [review]
I think it looks good! Just one minor change, plus my question about just doing the `iso2` change in this PR? Wondering, is there any way we should be testing these to make sure they don't silently fail eventually?

**Comment by turnerm** [comment]
> Wondering, is there any way we should be testing these to make sure they don't silently fail eventually?

Yes this will undoubtedly be an issue, the solution IMO would be some kind of E2E testing. But in the meantime we can just monitor the COD Skype group for any updates.

File: src/ochanticipy/config/countries/lby.yaml (hunk 1 above)
Comment (turnerm) [reply src/ochanticipy/config/countries/lby.yaml:7]: Oops thanks!

File: docs/config.rst [hunk 2]
```diff
@@ -15,12 +15,32 @@ and we are continuously adding more.
 
 .. _list of supported countries:
 
-A list of countries that we currently support:
+We currently support 20 out of 25 humanitarian response plan (HRP) countries:
```
Comment (caldwellst) [inline docs/config.rst:18]: I think here should we just add a caveat that not all data is available for all countries? Cause people might read it and expect their country to be there.

**Review by caldwellst — APPROVED** [review]
Just one question about adding a caveat that not all data sources available across all countries when listing "supported countries". Looks great!

**Pull request merged**